scripts/fix_readability_bulk.sh

```
@@ -0,0 +1,75 @@
+            echo -e "${BLUE}  ↳ Fixed pipeline in $(basename "$file")${NC}"
+        fi
+        rm -f "$file.bak"
+    fi
+done
+
+echo -e "\n${YELLOW}3. Removing parentheses from zero-arity function definitions...${NC}"
+# Fix patterns like: def config() -> def config
+find lib -name "*.ex" -type f | while read -r file; do
+    if sed -i.bak -E 's/def[[:space:]]+([[:alnum:]_]+)\(\)[[:space:]]*do/def \1 do/g' "$file" 2>/dev/null; then
+        if ! cmp -s "$file" "$file.bak"; then
+            FIXES=$((FIXES + 1))
```

Shellcheck (0.10.0)

[info] Modification of FIXES is local (to subshell caused by pipeline).

(SC2030)